src/libraries/System.Text.Json/tests/Serialization/DictionaryTests.cs

```diff
@@ -1637,15 +1637,16 @@ public static void DictionaryNotSupportedButIgnored()
             Assert.Null(obj.MyDictionary);
         }
 
+        // https://github.com/dotnet/runtime/issues/29933
         [Fact]
-        public static void Regression38643_Serialize()
+        public static void Serialize_WithCamelCase()
```

Comment:
I don't think the use of camel case in here is relevant to this test (at least it is not the primary purpose of it).
```suggestion
        public static void Serialize_DictionaryOfPoco()
```